gclient/src/api/calls.rs

```diff
@@ -204,6 +204,78 @@ impl GearApi {
             .await
     }
 
+    /// Migrates an active program identified by `src_program_id` onto another
+    /// node identified by `tgt_node_api` and returns the migrated program
+    /// identifier.
+    pub async fn migrate_program(
+        &self,
+        src_program_id: ProgramId,
+        tgt_node_api: &GearApi,
```

Comment:
Usually, we use pair `src`/`dest`.
```suggestion
        dest_node_api: &GearApi,
```